src/MediaManagerBundle/Resources/public/js/mediaManager.js

```diff
@@ -1014,7 +1014,13 @@
             var layover = self.openLayover(title, '');
 
             if (typeof height === 'undefined') {
-                height = layover.height() - layover.find('.title').height() - 50;
+
+                var height = layover.height();
```

Comment:
This should be sufficient: `height = $('#edit-container').height() - (layover.find('.title').height() - 50);`

The layover (opened with no html) will never have a useful height here.

Reply:
You are right. simplified the code.